servers/partner-ci/builders/common.prepare.sh

```diff
@@ -20,15 +20,16 @@ fi
 [ -z $ISO_FILE ] && (echo "ISO_FILE variable is empty"; exit 1)
 
 
-if [[ $ISO_FILE == *"Mirantis"* ]]; then
+if [[ $ISO_FILE == *"Mirantis"* ]] || [[ "${FUEL_RELEASE_VERSION}" == "10" ]] ; then
   export FUEL_RELEASE=$(echo $ISO_FILE | cut -d- -f2 | tr -d '.iso')
-fi
-if [[ $ISO_FILE == *"custom"* ]]; then
+else
   export FUEL_RELEASE=90
 fi
 
-if [[ "${UPDATE_MASTER}" == "true" ]] && [[ ${FUEL_RELEASE} != *"80"* ]]; then
+if [[ "${UPDATE_MASTER}" == "true" ]] && [[ ${FUEL_RELEASE} != *"80"* ]] && [[ ${FUEL_RELEASE} != *"100"* ]]; then
   [ ${SNAPSHOTS_ID} ] && export SNAPSHOTS_ID=${SNAPSHOTS_ID} || export SNAPSHOTS_ID=${CUSTOM_VERSION:10}
+elif [[ ${FUEL_RELEASE} == *"100"* ]]; then
+  export SNAPSHOTS_ID="from_iso"
```

Comment:
pay attention!!
there are some troubles with it
need to refactor testing-code
**2017-02-13 14:29:55 - INFO environment.py:237 -- Try to find environment '10.0.ui.released''**